src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.css

```diff
@@ -0,0 +1,3 @@
+.DraftEditor-root {
```

Comment:
this scares me a little bit, we use DraftEditor in more than 1 place, so a global change might have side effects. I'll dig in & see if we can accomplish it another way

Reply:
Yeah, I had the same reservation. I didn't spend more than 10 or so minutes looking into it though, so thanks for looking as well!